src/string_impls/from_string.rs

```diff
@@ -122,22 +121,22 @@ macro_rules! at_between {
 }
 
 /// Details about the error that occurred.
-#[derive(Debug, derive_more::Display, derive_more::From, PartialEq, Eq)]
+#[derive(Debug, PartialEq, Eq, thiserror::Error)]
 #[allow(missing_docs)]
 pub enum ParseErrorKind {
-    #[display("Expected a value")]
+    #[error("Expected a value")]
     ExpectedValue,
-    #[from]
-    Complex(ParseComplexError),
-    #[from]
-    Char(ParseCharError),
-    #[from]
-    String(ParseStringError),
-    #[from]
-    Number(ParseNumberError),
-    #[from]
-    BitSequence(ParseBitSequenceError),
-    #[from]
+    #[error(transparent)]
+    Complex(#[from] ParseComplexError),
+    #[error(transparent)]
+    Char(#[from] ParseCharError),
+    #[error(transparent)]
+    String(#[from] ParseStringError),
+    #[error(transparent)]
+    Number(#[from] ParseNumberError),
+    #[error(transparent)]
+    BitSequence(#[from] ParseBitSequenceError),
+    #[error("Custom error: {0}")]
     Custom(String),
```

Comment:
Maybe not important but no more `From<String>` impl :)

Reply:
lemme add it, missed that.